habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -372,8 +385,8 @@ def _reset_environment(self):
             self._remote_client_state.clear_history()
 
         # todo: fix duplicate calls to self.ctrl_helper.on_environment_reset() here
-        if self.ctrl_helper:
-            self.ctrl_helper.on_environment_reset()
+        # if self.ctrl_helper:
+        #     self.ctrl_helper.on_environment_reset()
```

Comment:
We'll probably want to uncomment this.

Reply:
I don't think we should uncomment this. There are two resets being called here, which was crashing episode initialization.

Reply:
Actually no, I don't think this is the 2x reset call. I think the other reset is happening on habitat-llm side that I need to disable.